src/main/java/mate/academy/spring/controller/DataInitializer.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.controller;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class DataInitializer {
+    private final UserService userService;
+    private final RoleService roleService;
+
+    public DataInitializer(UserService userService, RoleService roleService) {
+        this.userService = userService;
+        this.roleService = roleService;
+    }
+
+    @PostConstruct
+    public void inject() {
+        Role adminRole = new Role();
+        adminRole.setRole(Role.RoleName.ADMIN);
+        roleService.add(adminRole);
+        Role userRole = new Role();
+        userRole.setRole(Role.RoleName.USER);
+        roleService.add(userRole);
+        User user = new User();
+        user.setEmail("admin@i.ua");
+        user.setPassword("admin123");
+        user.setRoles(Set.of(adminRole));
+        userService.add(user);
+    }
+}
```

Comment:
sorry for the redundant comment, move it to `mate.academy.spring` and restore `Component` annotation